sections/gov/components/Proposal/Results.tsx

```diff
@@ -46,47 +50,51 @@ const Results: React.FC<ResultsProps> = ({ hash }) => {
 
 	if (proposal.isSuccess && proposal.data && choices) {
 		const { data } = proposal;
+
+		const totalVotes = data.totalVotesBalances !== 0 ? data.totalVotesBalances : 1;
+
+		const mappedResults = [] as any;
+
+		if (activeTab === SPACE_KEY.COUNCIL) {
+			data.totalBalances.forEach((balance, key) => {
+				mappedResults.push({
+					label: choices[key].name ? choices[key].name : choices[key].address,
+					balance: balance,
+				});
+			});
+		} else {
+			data.totalBalances.forEach((balance, key) => {
+				mappedResults.push({
+					label: choices[key],
+					balance: balance,
+				});
+			});
+		}
+
+		mappedResults.sort((a: any, b: any) => b.balance - a.balance);
```

Comment:
^ this part can be refactored to a bit cleaner

```
const mappedResults = data.totalBalances.map(...).sort(...)
```

The `activeTab === SPACE_KEY.COUNCIL` condition can be inside the map to differentiate)